orchestration/dagster_orchestration/hca_orchestration/pipelines/stage_data.py

```diff
@@ -37,4 +44,4 @@
     mode_defs=[prod_mode, local_mode, test_mode]
 )
 def stage_data() -> None:
-    pre_process_metadata(start=clear_staging_dir())
+    create_staging_dataset(pre_process_metadata(start=clear_staging_dir()))
```

Comment:
probably worth changing the keyword argument to a positional argument here, just for the sake of consistency